addons/loyalty/wizard/loyalty_history_wizard.py

```diff
@@ -0,0 +1,35 @@
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SaleLoyaltyHistoryWizard(models.TransientModel):
+    _name = 'sale.loyalty.history.wizard'
+    _description = 'History Coupons'
+
+    card_id = fields.Many2one(
+        comodel_name='loyalty.card',
+        required=True,
+        default=lambda self: self.env.context.get('active_id')
```

Comment:
```suggestion
        default=lambda self: self.env.context.get('active_id'),
```
same for other fields :see_no_evil: